helm-charts/falcon-sensor/templates/_helpers.tpl

```diff
@@ -63,6 +63,15 @@ Create the name of the service account to use
 
 {{- define "falcon-sensor.image" -}}
 {{- if .Values.node.enabled -}}
+{{ include "falcon-sensor-node.image" . }}
+{{- else -}}
+{{ include "falcon-sensor-container.image" . }}
+{{- end -}}
+{{- end -}}
+
+
+{{- define "falcon-sensor-node.image" -}}
```

Comment:
Shouldn't lines 64-72 now be dropped and daemonset/node instances of falcon-sensor.image be changed to "falcon-sensor-node.image"?